go.mod

```diff
@@ -26,6 +26,8 @@ require (
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.15.0
+	// Async ICQ branch: ibc-v4
+	github.com/strangelove-ventures/async-icq v0.0.0-20230116084035-5609e84dd443
```

Comment:
LGTM, can we add a release blocking TODO for this to be replaced with a tag?

Reply:
Added https://github.com/osmosis-labs/osmosis/issues/4218. I'll ping them to add a tag

Reply:
also, how do I make a ticket release blocking?

Reply:
We can add it to the tracker issue for now